api/registrations/serializers.py

```diff
@@ -344,6 +343,19 @@ def update(self, registration, validated_data):
                     raise exceptions.PermissionDenied()
             else:
                 raise exceptions.ValidationError('Registrations can only be turned from private to public.')
+        if validated_data.get('retraction') and not validated_data.get('is_retracted'):
+            raise exceptions.ValidationError(
+                'You cannot provide a withdrawal_justification without a concurrent withdrawal request.',
+            )
+        if is_truthy(validated_data.get('is_retracted', False)):
```

Comment:
Case handled where `is_retracted` in payload and value is True, but what if it was in payload and value is False?

My personal opinion, is if they try to set withdrawn to False, we throw an error saying they can't do that instead of ignoring it.